tests/test_external_strategies.py

```diff
@@ -0,0 +1,119 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from unittest.mock import MagicMock, Mock
+
+from d3a.models.area import Area
+from d3a.models.strategy.external_strategies.load import LoadForecastExternalStrategy
+from d3a.models.strategy.external_strategies.pv import PVForecastExternalStrategy
+from d3a_interface.constants_limits import GlobalConfig
+from pendulum import now, Duration
+from pytest import mark
+
+
+def ext_strategy_fixture(strategy):
+    config = MagicMock()
+    config.ticks_per_slot = 90
+    GlobalConfig.end_date = GlobalConfig.start_date + Duration(days=1)
```

Comment:
I fear that this would change the value of `GlobalConfig.end_date` for all the tests that are run after this. The reason is that at the moment this is not a fixture (in pytest's jargon) and there is no teardown method.